{
	"argument": "Аргумент <b>{0}</b>",
	"confirmed": "Подтвержденный",
	"expert": "Эксперт",
	"normal": "Нормальный",
	"lambda": "Лямбда",
	"leekscript": "Лукскрипт",
	"ls_function_definitions": "Функция \"{0}\" предлагает {1} определений.",
	"ls_function_information": "Информация о функции \"{0}\".",
	"ls_return_nothing": "ничего не возвращает",
	"lstype_boolean": "логический",
	"lstype_float": "плавающий",
	"lstype_int": "весь",
	"lstype_number": "число",
	"lstype_string": "цепь",
	"too_much_debug": "Достигнут лимит Debug() / mark()! (500 000 символов)",
	"class_name": "Имя класса",
	"class_super": "Супер класс класса",
	"class_fields": "Поля класса",
	"class_staticFields": "Статические поля класса",
	"class_methods": "Методы класса",
	"class_staticMethods": "Статические методы класса",
	"folder_renamed": "Папка переименована в {0}",
	"function_f": "Функция {0}",
	"ai_renamed": "ИИ переименован в {0}",
	"defined_in": "Определено в {0} в строке {1}",
	"keyword": "Ключевое слово {0}",
	"variable": "Переменная <b>{0}</b>",
	"variable_defined_in_ai": "Набор переменных в AI <b>{0}</b>",
	"chip_not_equipped": "Чип {0} ({1}) не оборудован",
	"chip_not_exists": "Чип {0} не существует",
	"no_weapon_equipped": "Без оружия",
	"default_name": "Без названия",
	"example_code": "//--------------------------------\n//------- Базовый код -----------\n//--------------------------------\n\n// Берем пистолет\nустановитьОружие(WEAPON_PISTOL); // Внимание: стоит 1 очко\n\n// Получаем ближайшего врага\nвар враг = getNearestEnemy();\n\n// Наступаем на врага\nдвигаться к врагу;\n\n// Мы пытаемся застрелить его!\nиспользовать оружие (враг);",
	"example_code_v2": "пусть делители = n -> [1..n.sqrt()].filter(x -> !(n % x))\nразделители (1989)",
	"random": "Случайный код",
	"error_1": "Невозможно использовать это имя функции",
	"error_2": "Невозможно использовать это имя параметра",
	"error_3": "Ожидаемая открывающая скобка",
	"error_4": "Ожидаемая открывающая скобка",
	"error_5": "Ожидалось имя параметра",
	"error_6": "Ожидалось имя функции",
	"error_7": "Закрывающая скобка ожидается после параметров функции",
	"error_8": "Не все блоки закрыты",
	"error_9": "Нет блоков для закрытия",
	"error_10": "Неожиданный конец файла",
	"error_11": "Здесь ожидался конец обучения",
	"error_12": "Ключевое слово break можно использовать только внутри цикла.",
	"error_13": "Ключевое слово continue можно использовать только внутри цикла.",
	"error_14": "Нельзя включать ИИ нигде, кроме основного блока",
	"error_15": "Здесь ожидается имя ИИ",
	"error_16": "ИИ {0} не существует",
	"error_17": "Ожидается закрывающая скобка",
	"error_18": "Ожидается закрывающая скобка",
	"error_19": "Невозможно объявить функцию нигде, кроме как в основном блоке",
	"error_20": "Ожидалось имя переменной",
	"error_31": "Это массив ключ: значение, невозможно объявить значение без его ключа",
	"error_36": "Невозможно добавить оператор, если в текущем блоке уже есть разрыв, возврат или продолжение.",
	"error_38": "Невозможно присвоить значение этому выражению",
	"error_39": "Несуществующая функция",
	"error_27": "Вы можете объявить глобальную переменную только в основном блоке",
	"error_61": "Неверный ИИ: {0}",
	"error_41": "Неправильный символ",
	"error_42": "Неправильный номер",
	"error_40": "Неверное количество параметров",
	"error_23": "Неожиданное ключевое слово",
	"error_24": "Неожиданное ключевое слово здесь",
	"error_26": "Нет условий для этого еще",
	"error_34": "Этот оператор здесь неожиданный",
	"error_32": "Открывающая скобка ожидается после ключевого слова function",
	"error_30": "Это однозначный массив, невозможно объявить значение с его ключом",
	"error_37": "Выражение не полное",
	"error_33": "Неизвестная переменная или функция \"{0}\"",
	"error_35": "Ожидалось значение",
	"error_22": "Переменная не существует",
	"error_29": "В имени переменной ожидается строка символов",
	"error_28": "Строка символов ожидается после глобального ключевого слова",
	"error_25": "Ожидается некоторое время после блока do",
	"error_63": "ИИ отключен, поэтому не запускается",
	"error_64": "Прерывание ИИ: {0}",
	"error_65": "Прерывание ИИ: выполнение ИИ заняло более 10 секунд, отключено до конца боя.",
	"error_58": "Невозможно выполнить \"{0}\"",
	"error_59": "Невозможно использовать эту функцию с аргументами ({0}), ожидаемые аргументы: {1}",
	"error_66": "Ваш ИИ не может быть правильно скомпилирован, потому что ваш код слишком велик.",
	"error_55": "Функция {0} будет удалена, теперь используйте {1}",
	"error_57": "Деление на ноль",
	"error_60": "У тебя нет ИИ",
	"error_56": "Неизвестная функция: {0}",
	"error_69": "Неизвестный метод \"{1}\" в классе \"{0}\"",
	"error_70": "Неизвестный статический метод \"{1}\" в классе \"{0}\"",
	"error_72": "Неизвестное поле \"{1}\" в классе \"{0}\"",
	"error_47": "Ключевое слово this разрешено только в нестатических конструкторах и методах.",
	"error_68": "Внутренняя ошибка, сообщение уже отправлено администратору",
	"error_48": "Это ключевое слово должно быть в классе",
	"error_51": "Неизвестный статический член \"{1}\" в классе \"{0}\"",
	"error_52": "Цепочка наследования этого класса зацикливается",
	"error_53": "Устарело: значения уже передаются по ссылке из LeekScript 2",
	"error_43": "Этот конструктор уже существует",
	"error_44": "Ожидается окончание занятий",
	"error_46": "Этот класс не существует",
	"error_49": "Ключевое слово супер здесь недоступно",
	"error_54": "Этот метод уже существует",
	"error_67": "Функция вашего ИИ не может быть правильно скомпилирована, потому что ее код слишком велик.",
	"error_71": "Метод string() должен возвращать строку",
	"error_73": "Неизвестный строитель",
	"error_74": "Второй оператор instanceof должен быть классом",
	"error_75": "Это значение \"{0}\" не повторяется (не является списком или таблицей).",
	"error_76": "Переполнение стека",
	"error_77": "Недопустимый оператор \"{0}\"",
	"error_78": "Это поле \"{1}\" является закрытым в классе \"{0}\".",
	"error_79": "Это поле \"{1}\" защищено в классе \"{0}\".",
	"error_80": "Это статическое поле \"{1}\" является закрытым в классе \"{0}\".",
	"error_81": "Это статическое поле \"{1}\" защищено в классе \"{0}\".",
	"error_82": "Этот метод \"{1}\" является закрытым в классе \"{0}\".",
	"error_83": "Этот метод \"{1}\" защищен в классе \"{0}\".",
	"error_84": "Этот конструктор является закрытым в классе \"{0}\".",
	"error_85": "Этот конструктор защищен в классе \"{0}\".",
	"error_86": "Этот статический метод \"{1}\" является закрытым в классе \"{0}\".",
	"error_87": "Этот статический метод \"{1}\" защищен в классе \"{0}\".",
	"error_91": "Это поле \"{0}\" зарезервировано.",
	"error_92": "Это значение \"{0}\" не является массивом или объектом.",
	"error_93": "Операторы === и !== устарели в LS 4+.",
	"error_94": "Эта функция удалена в LeekScript {0}, и вы находитесь в LeekScript {1}. См. энциклопедию, чтобы заменить его.",
	"error_95": "Эта функция доступна в LeekScript {0} и выше, и вы находитесь в LeekScript {1}.",
	"error_96": "Доступ к индексу {0} за пределами массива размером {1}.",
	"error_97": "Этот ключ дублируется в этой таблице.",
	"error_98": "Аргумент №{0} \"{1}\" типа \"{2}\" несовместим с ожидаемым типом \"{3}\".",
	"error_100": "Прерывание ИИ: недопустимое значение: {0}",
	"error_101": "Прерывание ИИ: слишком много операций.",
	"error_103": "Прерванный ИИ: потребляется слишком много оперативной памяти.",
	"error_104": "Допускается только один последовательный числовой разделитель.",
	"error_105": "Эта функция удалена в LeekScript {0}, и вы находитесь в LeekScript {1}. Он заменен на \"{2}\".",
	"error_106": "Запрещено переопределять функцию из LeekScript 4.",
	"error_107": "Невозможно назначить конечное поле.",
	"error_108": "Невозможно присвоить окончательное значение.",
	"error_109": "Дублирующий конструктор.",
	"error_110": "Ожидается стрела.",
	"error_111": "Невозможно изменить таблицу во время ее обхода.",
	"error_112": "Мертвая сущность",
	"error_114": "Эта переменная \"{0}\" назначается самой себе.",
	"error_115": "Это сравнение всегда ложно, так как типы \"{0}\" и \"{1}\" несовместимы.",
	"error_116": "Это сравнение всегда верно, поскольку типы \"{0}\" и \"{1}\" несовместимы.",
	"error_117": "Оператор '{1}{0}{2}' неизвестен.",
	"error_118": "Аргумент '{0}' дублируется.",
	"error_1000": "Нет снаряженного оружия. Сначала вам нужно экипировать оружие с помощью setWeapon(WEAPON_*).",
	"error_1001": "Этот чип {1} (идентификатор {0}) не оборудован. Вы должны сначала оборудовать его на своей странице лука-порея.",
	"error_1002": "Этот чип {0} не существует. Используйте константы CHIP_* для получения действительного чипа.",
	"error_1003": "Это оружие {0} не существует. Используйте константы WEAPON_* для получения действительного оружия.",
	"error_1004": "Это оружие {1} (идентификатор {0}) не экипировано. Вы должны сначала оборудовать его на своей странице лука-порея.",
	"error_1005": "У призванной лампочки нет ИИ",
	"error_119": "Невозможно присвоить значение \"{0}\" типа \"{1}\" переменной \"{2}\" типа \"{3}\".",
	"error_120": "Ожидаемый шеврон закрытия.",
	"error_121": "Ожидаемый тип.",
	"error_122": "Невозможно преобразовать тип \"{0}\" в тип \"{1}\".",
	"error_123": "Ожидаемая запятая.",
	"error_124": "Тип \"{0}\" не совместим с типом \"{1}\".",
	"error_125": "Опасное преобразование из \"{0}\" в \"{1}\".",
	"error_126": "Опасное преобразование \"{0}\" типа \"{1}\" в \"{2}\" типа \"{3}\".",
	"error_127": "Значение \"{0}\" типа \"{1}\" может не быть итерируемым.",
	"error_128": "Значение \"{0}\" типа \"{1}\" не может быть вызвано.",
	"error_129": "Значение \"{0}\" типа \"{1}\" может быть не вызываемым.",
	"error_130": "Невозможно преобразовать значение \"{0}\" типа \"{1}\" в тип \"{2}\".",
	"error_131": "Поле \"{0}\" может не существовать в значении \"{1}\" типа \"{2}\".",
	"error_132": "Бесполезное утверждение non-null, потому что значение \"{0}\" типа \"{1}\" никогда не является нулевым.",
	"error_133": "Значение \"{0}\" типа \"{1}\" может быть не индексируемым.",
	"error_134": "Значение \"{0}\" типа \"{1}\" не является индексируемым.",
	"error_135": "Этот метод типа \"{0}\" должен иметь тот же тип, что и его перегруженный родительский метод типа \"{1}\"",
	"error_136": "Нет необходимости преобразовывать \"{0}\" в \"{1}\".",
	"error_21": "Имя переменной \"{0}\" недоступно",
	"error_62": "Ваш ИИ не может быть скомпилирован правильно, администратор был уведомлен: {0}.",
	"error_50": "Этот член \"{0}\" не существует в \"{1}\" типа \"{2}\"",
	"error_45": "Это поле \"{0}\" уже существует",
	"error_99": "Неизвестная ошибка: {0}",
	"error_141": "Оператор {0} несовместим с \"{1}\" типа \"{2}\"",
	"error_142": "Бесконечная граница \"{0}\" не может быть замкнутой.",
	"error_137": "Слишком много ошибок",
	"error_143": "Ожидается оператор \"...\".",
	"error_144": "Незамкнутая символьная строка.",
	"error_145": "Аргументы по умолчанию должны располагаться в конце.",
	"error_139": "Невозможно использовать оператор in для этого значения \"{0}\"",
	"intial_version": "Первоначальная версия",
	"strict_mode": "Строгий режим (расширенный)",
	"all_info_ls": "Вся информация о {0}",
	"size": "Размер"
}
